{"name": "user tries to log in with different failing credentials -- @1.2 ", "status": "passed", "steps": [{"name": "Given I go to start page", "status": "passed", "start": 1609014944305, "stop": 1609014951454}, {"name": "When I click log in button", "status": "passed", "start": 1609014951455, "stop": 1609014953840}, {"name": "When I put my email \"FakeUser2\" and password \"<password>\"", "status": "passed", "start": 1609014953840, "stop": 1609014954001}, {"name": "And I confirm the log in", "status": "passed", "start": 1609014954001, "stop": 1609014956059}, {"name": "Then wrong email or password shows up", "status": "passed", "start": 1609014956059, "stop": 1609014956089}, {"name": "And close browser", "status": "passed", "start": 1609014956089, "stop": 1609014956121}], "parameters": [{"name": "username", "value": "FakeUser2"}, {"name": "passwor", "value": "FakePasswor"}], "start": 1609014944303, "stop": 1609014956122, "uuid": "b28b5840-4400-455a-aee9-2473b33694c5", "historyId": "0307e6cea5b31b18cd219a5a528d1872", "fullName": "features/login.feature:user tries to log in with different failing credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User logs"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}